pkgs/tools/X11/setroot/default.nix

```diff
@@ -5,14 +5,14 @@
 assert enableXinerama -> libXinerama != null;
 
 stdenv.mkDerivation rec {
-  version = "1.4.4";
+  version = "2017-01-24";
```

Comment:
Yes sorry should have mentioned that, forgot. Because the [latest commit](https://github.com/ttzhou/setroot/commit/16b07b3604708fb3ad7951a5f3743145fb450310) contains a bug fix. And it was commited nearly a year after v2.0 so it's unlikely there's going to be another new release soon, otherwise I would have just gone with v2.0.